content/10.diagnostics.md

```diff
@@ -41,7 +40,7 @@ This test shows a sensitivity range of 70-300 copies of the target RNA/ul.
 Furthermore, the assay needs very simple equipment and can be as fast as 30 minutes.
 Finally, the result indicator can be as simple as a color change on a strip [@doi:10.1126/science.abb8400].
 
-##### Limitation of Molecular Tests
+#### 4.1.d Limitation of Molecular Tests
 
 Molecular tests can detect only people that are currently infected, while they cannot detect someone who had developed the infection but has had since recovered.
```

Comment:
```suggestion
Molecular tests can only detect people that are currently infected, while they cannot detect someone who had developed the infection but has had since recovered.
```

Reply:
Does this depend on how we define "molecular" tests? I guess I'm not sure why a serological test wouldn't also be a "molecular" test. Perhaps something about "molecular" relates to testing for a nucleic acid associated with the virus. It might be good to call these "nucleic acid tests" at 4.1.